packages/concerto-tools/test/codegen/fromcto/jsonschema/jsonschemavisitor.js

```diff
@@ -215,7 +215,7 @@ describe('JSONSchema (samples)', function () {
             modelManager.addCTOModel( MODEL_BOUNDS );
             const visitor = new JSONSchemaVisitor();
             const schema = modelManager.accept(visitor, { rootType: 'test.Test'});
-            expect(schema.properties.myString.pattern).equal('^abc.*$');
+            expect(schema.properties.myString.pattern).equal('abc.*');
```

Comment:
This got looser? Is that what we want?

Reply:
Yes, I believe so. Previously we would always add these start/end tokens in the codegen, that modifies the declared regex from the model.
https://github.com/accordproject/concerto/blob/28304562ca32b15a39349f29c3c35e1ed66182b5/packages/concerto-tools/lib/codegen/fromcto/jsonschema/jsonschemavisitor.js#L299